composer.json

```diff
@@ -6,7 +6,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7.7",
+        "drupal/core": "^8.8",
```

Comment:
Are we finishing support of 8.7 officially?

Reply:
Yes, @ademarco proposed to do this. We are having issues with other dependencies that can't install on 8.7.